packages/orchestration/src/exos/orchestrator.js

```diff
@@ -161,6 +162,29 @@ export const prepareOrchestratorKit = (
     },
   );
 harden(prepareOrchestratorKit);
+
+/**
+ * @param {Zone} zone
+ * @param {{
+ *   asyncFlowTools: AsyncFlowTools;
+ *   chainHub: ChainHub;
+ *   localchain: Remote<LocalChain>;
+ *   chainByName: MapStore<string, HostInterface<Chain>>;
+ *   makeRecorderKit: MakeRecorderKit;
+ *   makeLocalChainFacade: MakeLocalChainFacade;
+ *   makeRemoteChainFacade: MakeRemoteChainFacade;
+ *   orchestrationService: Remote<CosmosInterchainService>;
+ *   storageNode: Remote<StorageNode>;
+ *   timerService: Remote<TimerService>;
+ *   vowTools: VowTools;
+ *   zcf: ZCF;
+ * }} powers
```

Comment:
Should we factor this out to an `OrchestratorPowers` typedef? The same definition seems to appear above on L47-L60

Reply:
I generally like the idea.  @turadg ?  

Note that since the type of one of those operations is derived from the other, it may be that one should just go away.